### DISCUSSION TITLE
Emails not forwarding from spamgourmet or mamber.net domains.

### DISCUSSION THREAD

**Discussion by greatwolf** [simple-login/app#1555] [Q&A]
I've already logged into spamgourmet.com and checked the alias counter I sent the test email to does decrement by one. So it appears emails is forwarding from their end but simple-login isn't picking it up for some reason.

Checking the simplelogin dashboard shows the last time I got an email from any of spamgourmet's domains was about a week ago. Any reason why this just seemly stopped working?

**Comment by amari077** [comment]
Hey greatwolf,
I have the same problem - many others also. This is a problem with spamgourmet. Check the discuss/support forum there.
greetings amari
